src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
@agcuda yes that is my suggestion. Otherwise I would think you would have to explain `ANONINNER` to the users. Explaining `null` seems easier to understand.